umbral/umbral.py

```diff
@@ -120,98 +119,142 @@ def __bytes__(self):
 
 
 class Capsule(object):
-    def __init__(self, point_eph_e, point_eph_v, bn_sig):
-        self.point_eph_e = point_eph_e
-        self.point_eph_v = point_eph_v
-        self.bn_sig = bn_sig
+    def __init__(self,
+                 point_eph_e=None,
+                 point_eph_v=None,
+                 bn_sig=None,
+                 e_prime=None,
+                 v_prime=None,
+                 noninteractive_point=None):
+
+        if not point_eph_e and not e_prime:
+            raise ValueError(
+                "Can't make a Capsule from nothing.  Pass either Alice's data (ie, point_eph_e) or Bob's (e_prime). \
+                Passing both is also fine.")
+
```

Comment:
bn_sig is also necessary

Reply:
OK, so you want the test to be for all three of either the original components or reconstructed components?

Reply:
Yes, that's perfect :)